index.js

```diff
@@ -0,0 +1,69 @@
+// global variables
+let searchCityForm = document.getElementById('searchCityForm');
+let errorMessage = document.getElementById('errorMessage');
+let city = document.getElementById('city');
```

Comment:
What element does `city` represent? Is it just the city name? Is it city and country? Be as specific as possible when naming